caas/kubernetes/provider/credentials.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package provider
+
+import (
+	"github.com/juju/errors"
+	"github.com/juju/juju/caas/kubernetes/clientconfig"
+
+	"github.com/juju/juju/cloud"
+	"github.com/juju/juju/environs"
+)
+
+const (
+	CredAttrUserName = "Username"
```

Comment:
Perhaps match upstream, "username" and "password". Also CredAttrUserName or CredAttrUsername ?

Reply:
CredAttrUserName was copied from elsewhere so I didn't change it